$NetBSD: patch-xpcom_reflect_xptcall_md_unix_moz.build,v 1.1 2022/05/01 00:36:10 manu Exp $

- NetBSD/macppc reports OS_TEST as powerpc and not macppc
- Use OpenBSD/powerpc MD stubs for NetBSD/powerpc

--- xpcom/reflect/xptcall/md/unix/moz.build.orig	2022-04-18 02:30:50.487997116 +0000
+++ xpcom/reflect/xptcall/md/unix/moz.build	2022-04-23 01:07:48.540922360 +0000
@@ -214,15 +214,16 @@
               'xptcstubs_asm_ppc64_linux.S',
               'xptcstubs_ppc64_linux.cpp',
           ]
 
-if CONFIG['OS_TEST'] in ('macppc', 'bebox', 'ofppc', 'prep', 'amigappc'):
+# OpenBSD/powerpc MD code is up to date and fits NetBSD
+if CONFIG['OS_TEST'] in ('powerpc', 'macppc', 'bebox', 'ofppc', 'prep', 'amigappc'):
     if CONFIG['OS_ARCH'] == 'NetBSD':
         SOURCES += [
-            'xptcinvoke_asm_ppc_netbsd.s',
-            'xptcinvoke_ppc_netbsd.cpp',
-            'xptcstubs_asm_ppc_netbsd.s',
-            'xptcstubs_ppc_netbsd.cpp',
+            'xptcinvoke_asm_ppc_openbsd.S',
+            'xptcinvoke_ppc_openbsd.cpp',
+            'xptcstubs_asm_ppc_openbsd.S',
+            'xptcstubs_ppc_openbsd.cpp',
         ]
 
 if CONFIG['OS_ARCH'] == 'OpenBSD' and CONFIG['OS_TEST'] == 'powerpc':
     SOURCES += [
